cvat/apps/engine/ddln/inventory_client.py

```diff
@@ -23,8 +23,8 @@ def record_sequence_completion(job_id, sequence_name, task_name, annotator, anno
 def record_task_creation(task, segments):
     if not segments:
         return
-    pairs = [(seq_name, (assignee.username if assignee else '')) for seq_name, _, _, _, assignee in segments]
     try:
+        pairs = [(seq_name, (assignee.username if assignee else '')) for seq_name, _, _, _, assignee in segments]
```

Comment:
Move this line into `try` block so task creation doesn't fail when an exception is raised here (generally, it doesn't raise, but might raise if I make changes in other code and forget to update this line.